.coveragerc

```diff
@@ -1,5 +1,12 @@
 [run]
+data_file = build/test/coverage/python_temp/.coverage
 omit =
     girder/external/*
     girder/test/*
 parallel = True
+
+[html]
+directory = build/test/artifacts/python_coverage
+
+[xml]
+output = build/test/coverage/python.xml
```

Comment:
What's with the name drift?  The standard name for this file is `coverage.xml`.  We had changed that to `server.xml`, and now to `python.xml`?

Reply:
This was after discussion with @danlamanna . `coverage.xml` is an ambiguous name, since we also have JS coverge that may be output in an XML format. The previous name of `server.xml` is also not properly descriptive, since this also includes coverage from the Python client.

At this point, I'm satisfied that this ought to be the final name. I'd chalk up the churn to normal development uncertainty and iteration.

Reply:
The think the ambiguity is debatable.  Nothing in the name `coverage.xml` implies python, and nothing in `python.xml` implies coverage.  Less ambiguous would be something like `py_coverage.xml` to match the `js_coverage.xml` file name from javascript.

Reply:
@manthey Good point. Done.